src/library_gl.js

```diff
@@ -945,7 +951,12 @@ var LibraryGL = {
         var glslVersion = GLctx.getParameter(GLctx.SHADING_LANGUAGE_VERSION);
         // extract the version number 'N.M' from the string 'WebGL GLSL ES N.M ...'
         var ver_re = /^WebGL GLSL ES ([0-9]\.[0-9][0-9]?)(?:$| .*)/;
-        var ver_num = glslVersion.match(ver_re);
+        var ver_num;
+#if WEBGL2_BACKWARDS_COMPATIBILITY_EMULATION
+        ver_num = '1.00'.match(/(.*)/);
+#else
+        ver_num = glslVersion.match(ver_re);
```

Comment:
may as well move the definition of `ver_re` into the else

Reply:
Done